src/utils.jl

```diff
@@ -189,6 +191,7 @@ end
 
 getname(buf::Vector{UInt8}) = "<raw buffer>"
 getname(cmd::Cmd) = string(cmd)
+getname(path::AbstractPath) = string(path)
 getname(str) = String(str)
```

Comment:
THis line was added in #394 to support FilePaths types.
Then FilePaths change from using `String` to usinmg `string` to get back to a plain string path type.
(which I still think is a dubious choice, but 🤷‍♂)

I take it is is not surficient anymore to  just do 
```suggestion
getname(str) = string(str)
```

So that means this line is probably not used anymiore
and so we can just:
```suggestion
```